src/js/components/reviewData/CertificationModal/ReviewDataCertifyModal.jsx

```diff
@@ -103,6 +108,11 @@ export default class ReviewDataCertifyModal extends React.Component {
 			hideClose = " hide";
 		}
 
+		let error = '';
+		if (this.state.errorMessage) {
+			error = <div className="alert alert-danger text-center" role="alert">{this.state.errorMessage}</div>;
+        }
```

Comment:
I thought i fixed this one but now it looks like it's too far out, i'll try again